ui/textual_summary.md

```diff
@@ -8,20 +8,37 @@ Hardware for Host etc.).
 
 Textual summaries consist of groups of related information and each group consist of properties.
 
-Definitions of these can be found in modules named according to scheme
+In a controller a list of textual groups is declared:
+
+```
+class CloudNetworkController < ApplicationController
+
+...
+
+  private
+
+  def textual_group_list
+    [%i(properties relationships), %i(tags)]
+  end
+  helper_method :textual_group_list
+end
+```
+
+Definition of each group can found in modules named according to scheme
 `XxxHelper::TextualSummary` in `app/helpers/*_helper/textual_summary.rb`.
 
 Example group definition:
 ```
-def textual_group_properties
-  %i(vapp aggregate_cpu_speed aggregate_cpu_memory aggregate_physical_cpus aggregate_cpu_total_cores aggregate_vm_memory aggregate_vm_cpus)
+def textual_group_power_management
+  TextualGroup.new(_("Power Management"), %i(power_state boot_time state_changed_on))
 end
 
-def textual_group_relationships
-  %i(parent_datacenter parent_cluster parent_host direct_vms allvms_size total_vms)
-end
 ```
 
+The group definition consists of a formater -- `TextualGroup`, a label and a list of properties.
+
+Other formaters include `TextualMultilabel` and `TextualTags`.
```

Comment:
Merging but @martinpovolny can you fix the `formater` to `formatter` in a followup PR in line 38 and 40?